nifi-commons/nifi-security-utils/src/main/java/org/apache/nifi/security/util/CertificateUtils.java

```diff
@@ -625,6 +634,20 @@ public static boolean isTlsError(Throwable e) {
         }
     }
 
+    /**
+     *Extracts the common name from the given DN.
+     *
+     * @param dn the distinguished name to evaluate
+     * @return the common name if it exists, null otherwise.
+     */
+    public static String getCommonName(String dn){
+        RDN[] rdns = new X500Name(dn).getRDNs(BCStyle.CN);
+        if(rdns.length == 0) {
```

Comment:
Recommend adding a space after the `if`:
```suggestion
        if (rdns.length == 0) {
```